common/app/views/main.scala.html

```diff
@@ -31,8 +31,7 @@
         @defining(
             page.item.content.trail.trailPicture.head
         ) { picture =>
-            <img src="http:@Html(ImgSrc.findNearestSrc(picture, Profile(width = Some(2200))).getOrElse(""))"
-            class="immersive-main-media__img">
+            @fragments.image(picture, imageAltText = "")
```

Comment:
I think we also need a `widths` value here, we'll probably have to create a new one?